browser/widevine/widevine_utils.cc

```diff
@@ -121,9 +112,89 @@ void InstallBundleOrRestartBrowser() {
                                    true);
   }
 }
+
+void SetWidevineInstalledVersion(const std::string& version) {
+  g_browser_process->local_state()->SetString(kWidevineInstalledVersion,
+                                              version);
+}
+
+std::string GetWidevineInstalledVersion() {
+  return g_browser_process->local_state()->GetString(kWidevineInstalledVersion);
+}
 #endif
 
+int GetWidevinePermissionRequestTextFrangmentResourceId() {
+  int message_id = -1;
+
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
+  message_id = IDS_WIDEVINE_PERMISSION_REQUEST_TEXT_FRAGMENT;
+#endif
+
+#if BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+  auto* manager = g_brave_browser_process->brave_widevine_bundle_manager();
+  message_id = manager->GetWidevinePermissionRequestTextFragment();
+#endif
+
+  DCHECK_NE(message_id, -1);
+  return message_id;
+}
+
+void RequestWidevinePermission(content::WebContents* web_contents) {
+  PermissionRequestManager::FromWebContents(web_contents)->AddRequest(
+      new WidevinePermissionRequest(web_contents));
+}
 void DontAskWidevineInstall(content::WebContents* web_contents, bool dont_ask) {
   Profile* profile = static_cast<Profile*>(web_contents->GetBrowserContext());
   profile->GetPrefs()->SetBoolean(kAskWidevineInstall, !dont_ask);
 }
+
+void RegisterWidevineProfilePrefsForMigration(
+    user_prefs::PrefRegistrySyncable* registry) {
+  registry->RegisterBooleanPref(kWidevineOptedIn, false);
+
+#if BUILDFLAG(BUNDLE_WIDEVINE_CDM)
```

Comment:
We don't need anything for `BUNDLE_WIDEVINE_CDM` in brave code. It does not apply to us and just makes the code more confusing